client/src/App.js

```diff
@@ -1,25 +1,28 @@
-import logo from './logo.svg';
+import { Routes, Route, Navigate, useNavigate } from 'react-router-dom';
+import AuthStore from './store/contexts/AuthStore';
+import Landing from './pages/Landing/Container/Landing';
+import Register from './pages/Landing/AuthForms/Register';
+import Login from './pages/Landing/AuthForms/Login';
+import Directions from './pages/Landing/Directions/Directions';
+import Home from './pages/Home/Home';
 import './App.css';
+import { useEffect } from 'react';
+
+export default function App() {
 
-function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Routes>
+        <Route path="*" element={<Navigate to={'/landing'}/>}/>
+        
+        <Route path="/landing" element={<AuthStore><Landing/></AuthStore>}>
```

Comment:
let's not name the route /landing. You can use a simple '/' or '/homepage' or '/dashboard'